analyzers/tests/SonarAnalyzer.UnitTest/Rules/AlwaysSetDateTimeKindTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using CS = SonarAnalyzer.Rules.CSharp;
+using VB = SonarAnalyzer.Rules.VisualBasic;
+
+namespace SonarAnalyzer.UnitTest.Rules;
+
+[TestClass]
+public class AlwaysSetDateTimeKindTest
+{
+    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.AlwaysSetDateTimeKind>();
+
+    [TestMethod]
+    public void AlwaysSetDateTimeKind_CS() =>
+        builderCS.AddPaths("AlwaysSetDateTimeKind.cs").Verify();
+
+    [TestMethod]
+    public void AlwaysSetDateTimeKind_VB() =>
+        new VerifierBuilder<VB.AlwaysSetDateTimeKind>().AddPaths("AlwaysSetDateTimeKind.vb").Verify();
+
+#if NET
```

Comment:
No need for `#if NET`. The test code under `AlwaysSetDateTimeKind_CSharp9` works under the .NET Framework as well.

Reply:
Since C#9 is not supported in .Net Framework.  We opted to run all C#9+ test cases only for .NET core